app/code/Magento/Theme/etc/di.xml

```diff
@@ -105,9 +105,6 @@
             <argument name="scope" xsi:type="const">Magento\Store\Model\ScopeInterface::SCOPE_STORE</argument>
         </arguments>
     </virtualType>
-    <type name="Magento\Framework\App\ActionInterface">
-        <plugin name="designLoader" type="Magento\Theme\Plugin\LoadDesignPlugin"/>
```

Comment:
The class `Magento\Theme\Plugin\LoadDesignPlugin` can probably removed alltogether after this change ?
As I see this was the only use of that class.

Reply:
Sure yes you can do that

Reply:
In fact we should disable or deprecated this file instead completely remove it for maintain backward compatible